docs/Deployment.md

```diff
@@ -252,3 +252,7 @@ Note that the environment variables that are required should be passed with the
 
 When finished, to stop the container, run:
 `docker stop ct-mollie-extension`
+
+## Prerequisites
```

Comment:
Suggest moving this up to the top of the file. Change `Node JS Runtime` title to `Prerequisites`, then merge the information